docs/contributors/code/release.md

```diff
@@ -4,41 +4,41 @@ The [Gutenberg repository](https://github.com/WordPress/gutenberg) on GitHub is
 
 Before you begin, there are some requirements that must be met in order to successfully release a stable version of the Gutenberg plugin. You will need to:
 
-- Be a member of the [Gutenberg development team](https://developer.wordpress.org/block-editor/block-editor/contributors/repository-management/#teams). This gives you the ability to launch the GitHub actions that are related to the release process and to backport pull requests (PRs) to the release branch.
-- Have write permissions on the [Make WordPress Core](http://make.wordpress.org/core) blog. This allows you to draft the release post.
-- Obtain approval from a member of the Gutenberg Core team in order to upload the new version Gutenberg to the WordPress.org plugin directory.
+-   Be a member of the [Gutenberg development team](https://developer.wordpress.org/block-editor/block-editor/contributors/repository-management/#teams). This gives you the ability to launch the GitHub actions that are related to the release process and to backport pull requests (PRs) to the release branch.
```

Comment:
There are many extra blank spaces between the markdown hyphen and the text. Are those expected? Do they change anything? It doesn't seem to affect the processed markdown result.

Reply:
It's the formatting that Prettier enforces. I have it enabled in my IDE, and it looks like we don't format markdown files in `docs` folder. It happens automatically for README files for packages, for instance. I think @Mamaduka applied the same set of changes in https://github.com/WordPress/gutenberg/commit/578ad3936c05e7699dd2464db7310df38bfdd36b. It's also why I need to rebase this PR, and most likely, most of the whitespace changes will disappear.

Reply:
I still see many changes and I confirmed it's coming from Prettier by executing:

<img width="804" alt="Screenshot 2023-08-18 at 11 15 11" src="https://github.com/WordPress/gutenberg/assets/699132/a1f5dd4b-558a-46a8-9402-979b7bf05616">

https://github.com/WordPress/gutenberg/pull/53762/commits/21a8d1c32cc9ca61fb275e2ab954e1104f5a5438 reflects the opinions that Prettier has.

Reply:
Thanks for confirming!